modules/apps/object/object-admin-rest-api/bnd.bnd

```diff
@@ -1,6 +1,6 @@
 Bundle-Name: Liferay Object Admin REST API
 Bundle-SymbolicName: com.liferay.object.admin.rest.api
-Bundle-Version: 11.0.1
+Bundle-Version: 11.1.0
```

Comment:
Hey @p-albuquerque, could you please move this bnd file change to another commit?

Reply:
Done